src/index.ts

```diff
@@ -23,6 +23,7 @@ async function main() {
   console.log({ eventName, sha, headSha, branch, owner, repo, GITHUB_RUN_ID });
   const token = core.getInput('access_token', { required: true });
   const workflow_id = core.getInput('workflow_id', { required: false });
+  const allow_matching_sha = core.getInput('allow_matching_sha', { required: true }) === 'true';
```

Comment:
```suggestion
  const ignore_sha = core.getInput('ignore_sha', { required: false }) === 'true';
```

Reply:
@styfle I've done similar for this options as the option `access_token`:
- `access_token` is `required: false` in `action.yml`,
- but is `required: true` in `index.ts` since it will always have a value (if not provided by the user, it will be `'${{ github.token }}'`)

From my tests, this works great since the option will always have a value in the TS code.

But yes, if you prefer, I can set it to false here as well 👍🏽